core/mobile/src/frontend/MobileAuthorizationFrontend.ts

```diff
@@ -0,0 +1,25 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module OIDC
+ */
+
+import { AccessToken } from "@itwin/core-bentley";
+import { AuthorizationClient } from "@itwin/core-common";
+import { MobileApp } from "./MobileApp";
+
+/** Utility to request OIDC/OAuth tokens from mobile frontend to mobile backend.
+ * @beta
+ */
+export class MobileAuthorizationFrontend implements AuthorizationClient {
+  protected _accessToken: AccessToken = "";
+
+  public async getAccessToken(): Promise<AccessToken> {
+    if (!this._accessToken) {
```

Comment:
This will also continue to return the token after it has expired. You can either not cache it at all (which would cause it to always call the backend), or you need to update the getAccessToken call to the backend to return both the token and the expiration date. If you cache (which I think you should), then as with the backend, if the current time is within 1 minute of the expiration date, fetch a new one.

Reply:
Fixed.